mopro-core/src/middleware/circom/mod.rs

```diff
@@ -510,4 +517,155 @@ mod tests {
+            "921117808165172908",
+            "7155116889028933260",
+            "16769940396381196125",
+            "17141182191056257954",
+            "4376997046052607007",
+            "17471823348423771450",
+            "16282311012391954891",
+            "70286524413490741",
+            "1588836847166444745",
+            "15693430141227594668",
+            "13832254169115286697",
+            "15936550641925323613",
+            "323842208142565220",
+            "6558662646882345749",
+            "15268061661646212265",
+            "14962976685717212593",
+            "15773505053543368901",
+            "9586594741348111792",
+            "1455720481014374292",
+            "13945813312010515080",
+            "6352059456732816887",
+            "17556873002865047035",
+            "2412591065060484384",
+            "11512123092407778330",
+            "8499281165724578877",
+            "12768005853882726493",
+          ];
+        let modulus = [
+            "13792647154200341559",
+            "12773492180790982043",
+            "13046321649363433702",
+            "10174370803876824128",
+            "7282572246071034406",
+            "1524365412687682781",
+            "4900829043004737418",
+            "6195884386932410966",
+            "13554217876979843574",
+            "17902692039595931737",
+            "12433028734895890975",
+            "15971442058448435996",
+            "4591894758077129763",
+            "11258250015882429548",
+            "16399550288873254981",
+            "8246389845141771315",
+            "14040203746442788850",
+            "7283856864330834987",
+            "12297563098718697441",
+            "13560928146585163504",
+            "7380926829734048483",
+            "14591299561622291080",
+            "8439722381984777599",
+            "17375431987296514829",
+            "16727607878674407272",
+            "3233954801381564296",
+            "17255435698225160983",
+            "15093748890170255670",
+            "15810389980847260072",
+            "11120056430439037392",
+            "5866130971823719482",
+            "13327552690270163501",
+          ];
+        let base_message = [
+            "18114495772705111902",
+            "2254271930739856077",
+            "2068851770",
+            "0",
+            "0",
+            "0",
+            "0",
+            "0",
+            "0",
+            "0",
+            "0",
+            "0",
+            "0",
+            "0",
+            "0",
+            "0",
+            "0",
+            "0",
+            "0",
+            "0",
+            "0",
+            "0",
+            "0",
+            "0",
+            "0",
+            "0",
+            "0",
+            "0",
+            "0",
+            "0",
+            "0",
+            "0",
+          ];
+
+        // let expected_output_vec = vec![0, 0, 0, 0, 0, 0, 0, 0, 32, 65, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 128, 171, 176, 96, 15, 67, 180, 73];
```

Comment:
I assume there's no specific output that we have to check to ensure validity, is this correct?

Reply:
Yes exactly!
The circuit is verifying the correctness of the RSA signature, but for privacy purposes, it's not outputting anything.